### PR TITLE
*: replace interface{} with Datum in index range

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

File: util/types/datum.go [hunk 1]
```diff
@@ -546,3 +549,13 @@ func DatumsToInterfaces(datums []Datum) []interface{} {
 	}
 	return ins
 }
+
+// MinNotNullDatum returns a datum represents minimum not null value.
+func MinNotNullDatum() Datum {
```
Comment (shenli) [inline util/types/datum.go]: Why not use a const?

Comment (coocood) [inline util/types/datum.go]: Datum can not be defined as constant. it doesn't compile.

**Comment by shenli** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**